index.js

```diff
@@ -3,19 +3,18 @@ const cors = require('cors');
 const { PokemonService } = require('./service');
 const swaggerUI = require('swagger-ui-express');
 const swaggerDoc = require('./swagger.json');
+const { PokemonController } = require('./controller');
+
+const pokemonService = new PokemonService();
+const allPokemons = new PokemonController(pokemonService);
 
 app.use(cors());
 
-app.use(swaggerUI.serve );
+app.use(swaggerUI.serve);
 app.get('/', swaggerUI.setup(swaggerDoc));
 
-
-app.get('/pokemons', async (req, res) => {
-	const pokemonService = new PokemonService();
-	const { pokemons, error } = await pokemonService.get();
-	if (error) return res.status(404).send(error);
-
-	return res.status(200).json(pokemons);
-})
+app
+	.route('/pokemons')
```

Comment:
you do this when you create a custom router. 
```suggestion
	.route('/pokemons')
app.get('/pokemons', pokemonController.getAllPokemons);
```